From edd63a5f7542e010fb44ff62146938bda80406f7 Mon Sep 17 00:00:00 2001
From: Manuel Fombuena <mfombuena@innovara.co.uk>
Date: Fri, 3 Nov 2023 11:53:38 +0000
Subject: [PATCH 09/10] fix(pcsc): add --disable-polkit to pcscd.service

Needs pcsc >= 2.0.0

polkit is not included in dracut and without this option, pcscd simply fails

(Cherry-picked commit 842be0c89fe0cc8cac9efa88e68b07ae1c2d2464 from PR#2547)
---
 modules.d/91pcsc/pcscd.service | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/91pcsc/pcscd.service b/modules.d/91pcsc/pcscd.service
index 639decd2..7d4e0a2b 100644
--- a/modules.d/91pcsc/pcscd.service
+++ b/modules.d/91pcsc/pcscd.service
@@ -5,7 +5,7 @@ Documentation=man:pcscd(8)
 Requires=pcscd.socket
 
 [Service]
-ExecStart=/usr/sbin/pcscd --foreground --auto-exit
+ExecStart=/usr/sbin/pcscd --foreground --auto-exit --disable-polkit
 ExecReload=/usr/sbin/pcscd --hotplug
 
 [Install]
-- 
2.42.0

